ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__aws-4.19-nightly-x86.yaml

```diff
@@ -88,7 +88,6 @@ tests:
         TicketId 532
       ZONES_COUNT: "3"
     test:
-    - ref: openshift-qe-workers-scale
```

Comment:
This step already checks if all the newly provisioned nodes are in `Ready` state. Followed by this, kube-burner's`cluster-health` checks if the all the [cluster operators](https://github.com/kube-burner/kube-burner-ocp/blob/2f7faae7c206930c27c7e0b02d4f0592345cc3d4/cluster_health.go#L62) are ready. Looks like we are missing something apart from these two checks. Keeping `openshift-qe-workers-scale` as is, can you try adding [this step](https://steps.ci.openshift.org/reference/openshift-qe-cluster-health) as cluster-health check which seems to be more reliable and re-run this experiment?